addons/stock/models/stock_rule.py

```diff
@@ -383,6 +383,7 @@ def _get_lead_days(self, product, **values):
         :return: the cumulative delay and cumulative delay's description
         :rtype: tuple[defaultdict(float), list[str, str]]
         """
+        env = self.env
```

Comment:
You could define `_` instead
```suggestion
        _ = self.env._
```